### ISSUE TITLE
Step to add family in opencga

### ISSUE THREAD

**Issue by Mohammedhusen** [opencb/opencga#1359]
Hi Team,

We could see the option to show family details, but how we can define family and individual and link them in opencga?

Appreciate if could share with us documents or detail steps to achieve this.

**Comment by pfurio** [comment]
Hi,

I guess you will be using version 1.4. I apologise but we don't have a cli in that version to create families and inviduals altogether. However, the web service is there and the functionality is implemented:
![image](https://user-images.githubusercontent.com/4201841/63764060-e356c700-c8bd-11e9-9a6a-ed9c986a7b4e.png)
Though the command line is not ready, you can still call to the WS using curl from bash:
```
curl -X POST --header "Content-Type: application/json" --header "Accept: application/json" --header "Authorization: Bearer myToken" -d "{
  \"id\": \"string\",
  ...
}" "http://bioinfodev.hpc.cam.ac.uk/opencga-test/webservices/rest/v1/families/create?study=myStudy"

```
Basically, you would need to replace _myToken_ for your actual token, _myStudy_ for your study id, the host url for your OpenCGA url and provide all the family contents within the brackets.
